### PR TITLE
reward calculation

### PR DESCRIPTION
**Pull request by JaredCorduan**
Adding the reward calculation to the spec.

### CODE REVIEW

File: latex/epoch.tex [hunk 1]
```diff
@@ -145,20 +159,6 @@ \subsection{Stake Distribution Calculation}
...
            \right\}
-      \nextdef
-      & \fun{totalStake} \in \powerset{\TxOut} \to (\Ptr \mapsto \HashKey) \to \Allocs\\
-      & ~~\to (\HashKey \mapsto \HashKey) \to \Allocs \to \Coin
-      & \text{total active stake} \\
-      & \fun{totalStake}~\var{outs}~\var{ptrs}~\var{stkeys}~\var{delegs}~\var{stpools} =
-        \sum_{\_\mapsto c\in \var{active}}c \\
-      & \where active = \activeStake{outs}{ptrs}{stkeys}{delegs}{stpools}
-      \nextdef
-      & \fun{stakeDistr} \in \powerset{\TxOut} \to (\Ptr \mapsto \HashKey) \to \Allocs\\
```
Comment (JaredCorduan) [inline latex/epoch.tex]: `totalStake` and `stakeDistr` are now a part of the calculation below.

File: latex/epoch.tex [hunk 2]
```diff
@@ -231,6 +226,146 @@ \subsection{Accounting Functions}
...
+      & \fun{maxPool} \in \PrtclConsts \to \Coin \to [0, 1] \to [0, 1] \to \Coin
+      & \text{max pool reward} \\
+      & \fun{maxPool}~\var{pc}~\var{R}~\sigma~\var{s}~\var{poolConsts} = \\
+      & ~~~\floor*{
+           \frac{R}{1 + a_0}
+           \cdot
+           \left(\sigma' + s'\cdot a_0\cdot\frac{\sigma' - s'\frac{z_0-\sigma'}{z_0}}{z_0}\right)} \\
+      & ~~~\where \\
+      & ~~~~~~~(a_0, k) = \fun{maxPoolConsts}~pc \\
+      & ~~~~~~~z_0 = 1/k \\
```
Comment (polinavino) [inline latex/epoch.tex]: Is maxPoolConsts and poolConsts defined anywhere?

File: latex/epoch.tex [hunk 3]
```diff
@@ -231,6 +226,146 @@ \subsection{Accounting Functions}
...
 \end{figure}
 
+%%
+%% Figure - Functions for the Reward Calculation
+%%
+\begin{figure}[htb]
+  \begin{align*}
+      & \fun{maxPool} \in \PrtclConsts \to \Coin \to [0, 1] \to [0, 1] \to \Coin
+      & \text{max pool reward} \\
+      & \fun{maxPool}~\var{pc}~\var{R}~\sigma~\var{s}~\var{poolConsts} = \\
```
[the remark this answers is not in the document]
Comment (JaredCorduan) [reply latex/epoch.tex]: ah, yes! `poolConsts` isn't needed, `a_0` and `k` come from the protocol constants now

File: latex/epoch.tex (hunk 2 above)
Comment (JaredCorduan) [reply latex/epoch.tex]: nope, good catch!

**Pull request merged**